qiskit/quantum_info/__init__.py

```diff
@@ -177,3 +182,4 @@
     decompose_clifford,
     two_qubit_cnot_decompose,
 )
+from .analysis import hellinger_distance, hellinger_fidelity, Z2Symmetries
```

Comment:
hellinger distance and fidelity are already imported at the top of this file 🙂

Reply:
my bad, for trying to fix merge conflicts directly on github